src/modules/aws_cloudwatch@0.0.1/index.ts

```diff
@@ -26,7 +26,7 @@ export const AwsCloudwatchModule: Module = new Module({
   mappers: {
     logGroup: new Mapper<LogGroup>({
       entity: LogGroup,
-      entityId: (e: LogGroup) => e?.logGroupName,
+      entityId: (e: LogGroup) => e.logGroupName ?? e.id,
```

Comment:
Seen a pattern in other modules, shouldn't this be `e.id.toString()`?

Reply:
Ahh, so I think it's actually not doing anything here. The output type *must* be a string, but since `logGroupName` is a required field, Typescript believes it'll never fall back to `e.id` and didn't complain about the mismatched type of it.